framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -6268,6 +6272,32 @@ VkResult VulkanReplayConsumerBase::OverrideGetAndroidHardwareBufferPropertiesAND
     }
 }
 
+// We want to allow skipping the query for tool properties because the capture layer actually adds this extension
+// and the application may end up using the query.  However, this extension may not be present for replay, so
+// we stub it out in that case.  This will generate warnings in the GfxReconstruct output, but it shouldn't result
+// in a failure.
+VkResult VulkanReplayConsumerBase::OverrideGetPhysicalDeviceToolProperties(
+    PFN_vkGetPhysicalDeviceToolProperties                         func,
+    VkResult                                                      original_result,
+    const PhysicalDeviceInfo*                                     physical_device_info,
+    PointerDecoder<uint32_t>*                                     pToolCount,
+    StructPointerDecoder<Decoded_VkPhysicalDeviceToolProperties>* pToolProperties)
+{
+    const auto& instance_extensions = physical_device_info->parent_enabled_extensions;
+    if (std::find(instance_extensions.begin(), instance_extensions.end(), VK_EXT_TOOLING_INFO_EXTENSION_NAME) !=
+        instance_extensions.end())
+    {
+        return func(physical_device_info->handle, pToolCount->GetOutputPointer(), pToolProperties->GetOutputPointer());
+    }
+    else
+    {
+        GFXRECON_LOG_INFO("Skipping call down to vkGetPhysicalDeviceToolProperties because extension not supported by "
```

Comment:
To be consistent with other messages in `VulkanReplayConsumerBase`, I suggest phrasing this as `The captured application used vkGetPhysicalDeviceToolProperties. The is not supported by the replay device, so replay may fail.`